easybuild/easyconfigs/n/napari/napari-0.4.15-foss-2021a.eb

```diff
@@ -0,0 +1,161 @@
+    }),
+    ('qtconsole', '5.3.1', {
+        'checksums': ['b73723fac43938b684dcb237a88510dc7721c43a726cea8ade179a2927c0a2f3'],
+    }),
+    ('wrapt', '1.12.1', {
+        'checksums': ['b62ffa81fb85f4332a4f609cab4ac40709470da05643a082ec1eb88e6d9b97d7'],
+    }),
+    ('cachey', '0.2.1', {
+        'checksums': ['0310ba8afe52729fa7626325c8d8356a8421c434bf887ac851e58dcf7cf056a6'],
+    }),
+    ('docstring-parser', '0.14.1', {
+        'sources': 'docstring_parser-%(version)s.tar.gz',
+        'checksums': ['2c77522e31b7c88b1ab457a1f3c9ae38947ad719732260ba77ee8a3deb58622a'],
+    }),
+    ('pep517', '0.12.0', {
+        'checksums': ['931378d93d11b298cf511dd634cf5ea4cb249a28ef84160b3247ee9afb4e8ab0'],
+    }),
+    ('tomli', '2.0.1', {
+        'checksums': ['de526c12914f0c550d15924c62d72abc48d6fe7364aa87328337a31007fe8a4f'],
+    }),
+    ('magicgui', '0.5.1', {
+        'checksums': ['dcd05c4c9bbd55bdeb72d41069b2d4304952dabe04c2aac7bdd43f5a4e3a8a94'],
+    }),
+    ('napari-console', '0.0.4', {
+        'modulename': 'napari_console',
+        'checksums': ['ea0e30fbfdd76542adda44be457f1306a65146fe7578f95838e28d62874f4c45'],
+    }),
+    ('napari-plugin-engine', '0.2.0', {
+        'checksums': ['fa926f869d70e0d652c005661948cd0c7fee5508ae17d437937f34f5287590b3'],
+    }),
+    ('napari-svg', '0.1.6', {
+        'checksums': ['12fa5c64995fc7179349969a39316355496b463858c3b0815e3b3e7ffcdced34'],
+    }),
+    ('Pint', '0.19.2', {
+        'checksums': ['e1d4989ff510b378dad64f91711e7bdabe5ca78d75b06a18569ac454678c4baf'],
+    }),
+    ('build', '0.8.0', {
+        'checksums': ['887a6d471c901b1a6e6574ebaeeebb45e5269a79d095fe9a8f88d6614ed2e5f0'],
+    }),
+    ('psygnal', '0.3.5', {
+        'patches': ['psygnal-0.3.5_python3.8.patch'],
+        'checksums': [
+            '5abf84f0da9f487be68299854c4eb39de0765236407736d201f97ef39913a548',  # psygnal-0.3.5.tar.gz
+            'c55cc07c646cb66e8c5d2ea227249216dbd1a1a9405acbd330d70105a3fd8890',  # psygnal-0.3.5_python3.8.patch
+        ],
+    }),
+    ('pydantic', '1.9.1', {
+        'checksums': ['1ed987c3ff29fff7fd8c3ea3a3ea877ad310aae2ef9889a119e22d3f2db0691a'],
+    }),
+    ('typer', '0.4.1', {
+        'checksums': ['5646aef0d936b2c761a10393f0384ee6b5c7fe0bb3e5cd710b17134ca1d99cff'],
+    }),
+    ('npe2', '0.5.0', {
+        'checksums': ['3d3d43b99bbd3b189fedd7313589c89642a4af0f14b24b9763eb134ba6a5548c'],
+    }),
+    ('numpydoc', '1.4.0', {
+        'checksums': ['9494daf1c7612f59905fa09e65c9b8a90bbacb3804d91f7a94e778831e6fcfa5'],
+    }),
+    ('QtPy', '2.1.0', {
+        'checksums': ['ca8cd4217175186344299ee4c0f7e7adcf362c70852ba35b255a534077025c06'],
+    }),
+    ('superqt', '0.3.2', {
+        'checksums': ['a37df244a2d6e4387e865cdedabe925798b3352ab9cdc52f60701eb12d907960'],
+    }),
+    ('tifffile', '2022.5.4', {
+        'checksums': ['b03147a15862b7c1d90d47435197f149bef7a52c25ad67cf1f9b465faa71b8d2'],
+    }),
+    ('pandas', '1.3.0', {
```

Comment:
`pandas` is part of `SciPy-bundle`

Reply:
@jfgrimm  SciPy-bundle/2021.05-foss-2021a provides pandas 1.2.4. napari requires pandas >= 1.3.0 for Python >= 3.9

Reply:
could we bump everything to to `2021b`? I'm not a huge fan of duplicating most of the `SciPy-bundle` extensions within another easyconfig

Reply:
@jfgrimm yes, makes sense. I am working on it.
What would you recommend regarding the missing pyrcc5 tool in PyQt5? (see added PyQt5/PyQt5-5.15.4-GCCcore-10.3.0-tools.eb)

Reply:
I would think that change would be sensible in the main PyQt5 easyconfig, no need to separate it with a versionsuffix, since it should only be adding functionality?

Reply:
> could we bump everything to to `2021b`? I'm not a huge fan of duplicating most of the `SciPy-bundle` extensions within another easyconfig

@jfgrimm done